test/cli/template-use/common-dir/out/project/.keboola/manifest.json

```diff
@@ -33,7 +33,10 @@
   "branches": [
     {
       "id": 12345,
-      "path": "main"
+      "path": "main",
+      "metadata": {
+        "KBC.KAC.templates.instances": "[{\"instanceId\":\"%s\",\"templateId\":\"my-template-id\",\"version\":\"1.0.0\"}]"
```

Comment:
dopln prosim sem: https://keboola.atlassian.net/wiki/spaces/KB/pages/85491820/Storage+Metadata#Configuration-metadata-keys-(added-in-Q4%2F2021)

Reply:
(len to bude nejaka samostatna sekcia `branch metadata`)

Reply:
Jasně, hotovo.